_benchmark/reference/workloads/test-procedures.md

```diff
@@ -0,0 +1,193 @@
+`target-interval` | No | Interval | Defines an internal of less 1 / target-throughput (in seconds) less than one operation per second. Define either target-throughput or target-interval but not both (otherwise Rally will raise an error).
+`ignore-response-error-level` | No | Boolean | Controls whether to ignore errors encountered during the task when a benchmark is run with the `on-error=abort` command flag. 
+
+### Parallel tasks
+
+The `parallel` element runs tasks wrapped inside the element concurrently. 
+
+When running tasks in parallel, each task requires the `client` option to make sure clients inside your benchmark are reserved for that task. Otherwise, when the client option is specified inside the parallel element without a connection to the task, the benchmark will use that number of clients for all tasks.
+
+#### Usage
+
+In the following example, `parallel-task-1` and `parallel-task-2` execute a `bulk` operation concurrently:
+
+```yml
+{
+  "name": "parallel-any",
+  "description": "Track completed-by property",
+  "schedule": [
+    {
+      "parallel": {
+        "tasks": [
+          {
+            "name": "parellel-task-1",
+            "operation": {
+              "operation-type": "bulk",
+              "bulk-size": 1000
+            },
+            "clients": 8
+          },
+          {
+            "name": "parellel-task-2",
+            "operation": {
+              "operation-type": "bulk",
+              "bulk-size": 500
+            },
+            "clients": 8
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+#### Options
+
+The `parallel` element supports all `schedule` parameters, in addition to the following:
+
+`tasks` | Yes | Array | Defines a list of tasks that should be executed concurrently. 
+`completed-by` | No | String | Allows you define the name of one task in the tasks list, or the value `any`. If a specific task name has been provided then as soon as the named task has completed, the whole parallel task structure is considered completed. If the of value `any` is provided, then any task that completes first renders all other tasks specified in parallel structure complete. If this property is not explicitly defined, the parallel task structure is considered completed as soon as the tasks in the element complete.
+
+### Iteration-based options
+
+Iteration-based options determine the number of times an operation should fun. It can also define the number of iterative runs when tasks are run in [parallel](#parallel-tasks). To configure an iteration-based schedule, use the following options.
```

Comment:
```suggestion
Iteration-based options determine the number of times an operation should run. It can also define the number of iterative runs when tasks are run in [parallel](#parallel-tasks). To configure an iteration-based schedule, use the following options.
```